parts/k8s/kubernetesconfigs.sh

```diff
@@ -196,9 +196,10 @@ configureCNI() {
     retrycmd_if_failure 120 5 25 modprobe br_netfilter || exit $ERR_MODPROBE_FAIL
     echo -n "br_netfilter" > /etc/modules-load.d/br_netfilter.conf
     if [[ "${NETWORK_PLUGIN}" = "azure" ]]; then
-        if [[ "${NETWORK_POLICY}" != "calico" ]]; then
-            mv $CNI_BIN_DIR/10-azure.conflist $CNI_CONFIG_DIR/
-            chmod 600 $CNI_CONFIG_DIR/10-azure.conflist
+        mv $CNI_BIN_DIR/10-azure.conflist $CNI_CONFIG_DIR/
+        chmod 600 $CNI_CONFIG_DIR/10-azure.conflist
+        if [[ "${NETWORK_POLICY}" == "calico" ]] || [[ "${NETWORK_POLICY}" == "" ]]; then
+          sed -i 's#"mode":"bridge"#"mode":"transparent"#g' $CNI_CONFIG_DIR/10-azure.conflist
```

Comment:
Do we really want to change the Azure CNI configuration here? This bridge mode setting will now apply to non-calico scenarios given the `|| [[ "${NETWORK_POLICY}" == "" ]]` condition

Reply:
The purpose for this change is to make life easier for users who want to apply calico for network policy on clusters previously installed without specify a network policy provider. 
My understanding is it should not affect any user experience if they choose a non-calico network policy provider. I am not sure if user really care about using a `bridge` mode network or a `transparent` mode network. Would love to hear your input on this. Thanks.

Reply:
@sharmasushant @tamilmani1989 can you comment on this change?

Reply:
By default we should have bridge mode and not transparent mode. To my understanding, the config should be set to transparent mode only if user speicifes calico policy. @sharmasushant  can comment on this.

Reply:
@tamilmani1989 I'm running this through some final smoke tests. Please comment if we still have open concerns about this change before merging.